scripts/build/ci-build/Dockerfile

```diff
@@ -35,6 +35,8 @@ RUN apt-get update   && \
         curl            \
         git             \
         make            \
+        cmake           \
```

Comment:
Just to avoid errors?

Reply:
Required to build

Reply:
So weird that it worked for me first time. I guess some of my local containers was cached.